### PR TITLE
ENH: Add CDC county vaccine data

### PR DESCRIPTION
**Pull request by cc7768**
@ghop02 @sglyon 

Popping in to add to your review load.

### CODE REVIEW

**Review by sglyon — APPROVED** [review]

File: can_tools/scrapers/base.py [hunk 1]
```diff
@@ -37,9 +37,17 @@ def __init__(
         self.sex = sex
 
 
+class RequestError(Exception):
```
Comment (ghop02) [inline can_tools/scrapers/base.py:40]: suggestion/thought for the future: as long as there are more than a few I've always liked having an `errors` module that makes it easy to reference common errors across different contexts.

File: can_tools/scrapers/official/federal/CDC/cdc_county_vaccines.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            age="65_plus",
+        ),
+    }
+
+    def fetch(self):
+        response = requests.get(self.url)
+        if not response.ok:
+            msg = f"Failed to make request to {self.url}\n"
+            msg += "Response from request was\n:"
+            msg += textwrap.indent(response, "\t")
```
Comment (ghop02) [inline can_tools/scrapers/official/federal/CDC/cdc_county_vaccines.py:55]: Should this be `response.content`?

File: can_tools/scrapers/variables.py [hunk 3]
```diff
@@ -2,31 +2,31 @@
 
 from can_tools.scrapers import CMU
 
-INITIATING_VACCINATIONS_ALL = CMU(
+INITIATING_VACCINATIONS_ALL: CMU = CMU(
```
Comment (ghop02) [inline can_tools/scrapers/variables.py:5]: Is this necessary when the variable is directly assigned?

**Review by ghop02 — APPROVED** [review]
Couple small comments!

File: can_tools/scrapers/base.py (hunk 1 above)
Comment (sglyon) [reply can_tools/scrapers/base.py:40]: Yeah that works for me. so far I've just been collecting them here

File: can_tools/scrapers/official/federal/CDC/cdc_county_vaccines.py (hunk 2 above)
Comment (sglyon) [reply can_tools/scrapers/official/federal/CDC/cdc_county_vaccines.py:55]: yep, good catch!

File: can_tools/scrapers/variables.py (hunk 3 above)
Comment (sglyon) [reply can_tools/scrapers/variables.py:5]: For some reason my editor wasn't picking up that it was a CMU instance when I was using `variables.INITIATING_VACCINATIONS_ALL`

I agree it shouldn't be necessary, but I guess for my tooling it was?

**Comment by ghop02** [comment]
@sglyon is this good to go?

**Pull request merged**